src/pkg/control/restore.go

```diff
@@ -121,9 +121,9 @@ func (rc RestoreConfig) marshal() string {
 func (rc RestoreConfig) concealed() RestoreConfig {
 	return RestoreConfig{
 		OnCollision:        rc.OnCollision,
-		ProtectedResource:  clues.Hide(rc.ProtectedResource).Conceal(),
+		ProtectedResource:  clues.Conceal(rc.ProtectedResource),
 		Location:           path.LoggableDir(rc.Location),
-		Drive:              clues.Hide(rc.Drive).Conceal(),
```

Comment:
Missing context on the relevance of this `Hide` function. Can you explain where the 'loop' in the description is?

Reply:
The loop is below in `Format` I believe.

Reply:
Ah, this change is incidental.  The loop is on line 141.  `rc.concealed()` returns another restore config.  `fmt.Fprint`, when handed a struct, internally calls `v.Format(fs, r)`, which hits the Format func again, and thus loops.  So calling marshal before Fprint ensures we pass in a string instead of a struct with a Format method.

Reply:
@ryanfkeepers - do we need the change on line 124/125 above? Are we guaranteed that the items we are trying to conceal have already been hidden i.e. `hashText` has been generated?

Reply:
the 124/126 changes are a more direct version of the same call.  Same result either way.

Reply:
Got it - I re-read the code. Makes sense.